tests/e2e/connectivity/basic_test.go

```diff
@@ -30,9 +31,11 @@ var (
 	gwctl2             *client.Client
 )
 
+var cpType = flag.String("controlplane", "old", "Check which control-plane to use")
```

Comment:
Do we need this? we do not plan to keep the old control plane or make any changes on it. Why not keep the tests only the new control plane only.

Reply:
yes I will remove it when we decide to remove the old controlplane